packages/docusaurus-plugin-openapi/src/index.ts

```diff
@@ -274,6 +274,8 @@ export default function pluginOpenAPI(
                     rehypePlugins,
                     beforeDefaultRehypePlugins,
                     beforeDefaultRemarkPlugins,
+                    // Docusaurus 2.2.0 has a regression that requires this option to be set.
+                    markdownConfig: {},
```

Comment:
I would rather pass `markdownConfig: context.siteConfig.markdown`: this is what all other plugins do, and third-party plugin authors using our mdx loader should rather do the same to apply global markdown config for all md docs. There will be more Markdown site options in the future